ogr/ogrsf_frmts/flatgeobuf/ogrflatgeobufdataset.cpp

```diff
@@ -127,6 +127,12 @@ void RegisterOGRFlatGeobuf()
     GDALDriver *poDriver = new GDALDriver();
     poDriver->SetDescription("FlatGeobuf");
     poDriver->SetMetadataItem(GDAL_DCAP_VECTOR, "YES");
+    poDriver->SetMetadataItem(GDAL_DCAP_CREATE_LAYER, "YES");
+    poDriver->SetMetadataItem(GDAL_DCAP_CREATE_FIELD, "YES");
+    poDriver->SetMetadataItem(GDAL_DCAP_DELETE_FIELD, "YES");
+    poDriver->SetMetadataItem(GDAL_DCAP_REORDER_FIELDS, "YES");
+    poDriver->SetMetadataItem(GDAL_DCAP_CURVE_GEOMETRIES, "YES");
```

Comment:
to remove per above comment

Reply:
@rouault  was this comment supposed to be directed at the filegdb or openfilegdb drivers install of flatgeobuf?